### PR TITLE
Fix other tests broken by html changes

### PR DESCRIPTION
**Pull request by bdattoma**
Fixing some test cases broken by HTML changes:
- ODS-488
- ODS-1226
- ODS-327
- ODS-271 (was broken in Smoke for self-managed RHODS)
- ODS-1166
- ODS-1305
- ODS-489

The above test cases are only examples, there should be more affected by this PR

### CODE REVIEW

**Review by jgarciao — APPROVED** [review]
lgtm

**Comment by bdattoma** [comment]
Jenkins job on RHODS v1.23 (PSI cluster): [1580](https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1580/TestReport/): the failures are due to the RHOSi Setup (no changes due to the PR) and broken links in the product for ODS-488. The changes done by this PR are working fine.

Jenkins job on RHODS v1.22 (OSD cluster): [1582](https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1582/console) _RUNNING_

File: tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot [hunk 1]
```diff
@@ -145,26 +146,28 @@ Verify Service Is Available In The Explore Page
   Menu.Navigate To Page    Applications    Explore
   Wait For RHODS Dashboard To Load    expected_page=Explore
   Capture Page Screenshot
-  IF    ${split_last} == ${TRUE}
+  ${version_check}=  Is RHODS Version Greater Or Equal Than  1.23.0
+  IF    "${split_last}"=="${TRUE}" and "${version_check}"=="True"
```
Comment (lugi0) [inline tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot:150]: My understanding is that we shouldn't use these version checks anymore, see https://github.com/red-hat-data-services/ods-ci/pull/711#discussion_r1131822042

Comment (bdattoma) [reply tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot]: done

Comment (lugi0) [reply tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot]: Yep, that was news to me as well. Guess we could talk more in the next automation meeting.

**Review by lugi0 — APPROVED** [review]

**Review by jgarciao — APPROVED** [review]
lgtm

**Pull request merged**